### PR TITLE
Topbar update

### PR DESCRIPTION
**Pull request by mikkio-j**
Issue: #391

### CODE REVIEW

File: src/components/PublishingTopbar/PublishingTopbar.tsx [hunk 1]
```diff
@@ -103,7 +104,19 @@ const PublishingTopbar: React.FC = () => {
       <StyledTopbarBase variant="studio">
         <StudioContainer>
```
Comment (drillprop) [inline src/components/PublishingTopbar/PublishingTopbar.tsx]: I know that is not related to this task, but I really don't like that name. We have already `StudioContainer` and it's responsible for studio layout.  I wrongly imported this guy once 😅 
Could you rename this component in this PR?

File: src/components/PublishingTopbar/PublishingTopbar.tsx [hunk 2]
```diff
@@ -155,27 +168,31 @@ const NavDrawer = React.forwardRef<HTMLDivElement, NavDrawerProps>(
...
+        {hasChannels && (
+          <StyledLink to={routes.studioNewChannel()} onClick={handleClose}>
+            <NewChannel>
+              <NewChannelIconContainer>
+                <Icon name="new-channel" />
+              </NewChannelIconContainer>
+              <Text>New Channel</Text>
+            </NewChannel>
+          </StyledLink>
+        )}
```
Comment (drillprop) [inline src/components/PublishingTopbar/PublishingTopbar.tsx]:
```suggestion
        {hasChannels && (
          <>
            <Text variant="h6">My Channels</Text>
            {channels.map((channel) => (
              <ChannelInfo
                key={channel.name}
                channel={channel}
                member={memberName}
                active={channel.name === currentChannel.name}
                onClick={() => onCurrentChannelChange(channel)}
              />
            ))}
            <StyledLink to={routes.studioNewChannel()} onClick={handleClose}>
              <NewChannel>
                <NewChannelIconContainer>
                  <Icon name="new-channel" />
                </NewChannelIconContainer>
                <Text>New Channel</Text>
              </NewChannel>
            </StyledLink>
          </>
        )}
```

File: src/components/Topbar/StudioTopbar/StudioTopbar.style.tsx [hunk 3]
```diff
@@ -31,9 +32,13 @@ export const ChannelInfoContainer = styled.div<CommonStudioTopbarProps>`
...
-  svg {
-    margin-left: auto;
-    margin-right: ${sizes(3)};
+  button {
+    margin: 0 ${sizes(2)} 0 ${sizes(5)};
+    opacity: 0.4;
+    svg {
+      width: 20px;
+      height: 20px;
+    }
```
Comment (kdembler) [inline src/components/Topbar/StudioTopbar/StudioTopbar.style.tsx]: Which button is this styling?

File: src/components/Topbar/StudioTopbar/StudioTopbar.tsx [hunk 4]
```diff
@@ -101,11 +103,30 @@ const StudioTopbar: React.FC = () => {
...
+            <ChannelInfo channel={currentChannel} member={member.name} onClick={handleDrawerToggle} />
+          ) : (
+            <>
+              <NewChannelButtonLink
+                icon="new-channel"
+                variant="tertiary"
+                size="large"
+                to={routes.studio.newChannel()}
+              />
+              <StyledLink to={routes.studio.newChannel()}>
```
Comment (kdembler) [inline src/components/Topbar/StudioTopbar/StudioTopbar.tsx]: We have both the `ButtonLink` and `StyledLink` here, do we need both?

Comment (kdembler) [reply src/components/Topbar/StudioTopbar/StudioTopbar.tsx]: I also don't think this needs to be a link. If there are no channels, we will force the user go through the create channel flow

File: src/components/Topbar/StudioTopbar/StudioTopbar.style.tsx [hunk 5]
```diff
@@ -108,6 +108,7 @@ export const MemberTitleText = styled(Text)`
 
 export const StyledLink = styled(Link)`
   text-decoration: none;
+  color: white;
```
Comment (kdembler) [inline src/components/Topbar/StudioTopbar/StudioTopbar.style.tsx]:
```suggestion
  color: ${colors.white};
```

File: src/components/Topbar/StudioTopbar/StudioTopbar.style.tsx [hunk 6]
```diff
@@ -59,18 +59,11 @@ export const TextContainer = styled.div`
...
   display: flex;
   align-items: center;
   margin-left: auto;
   margin-right: ${sizes(3)};
-  svg {
-    width: auto;
-    height: auto;
-  }
-  button {
-    cursor: pointer;
```
Comment (kdembler) [inline src/components/Topbar/StudioTopbar/StudioTopbar.style.tsx:72]: Removing this makes the `pointer` go away. Can you look into the `Button` component and see why it doesn't apply this style?

**Review by kdembler — APPROVED** [review]

**Pull request merged**